concordium-consensus/src/Concordium/Scheduler/Environment.hs

```diff
@@ -348,7 +348,11 @@ class (StaticInformation m, ContractStateOperations m, MonadProtocolVersion m) =
     -- keep track of changes locally first, and only commit them at the end.
     -- Instance keeps track of its own address hence we need not provide it
     -- separately.
-    withInstanceStateV1 :: UInstanceInfoV m GSWasm.V1 -> UpdatableContractState GSWasm.V1 -> (ModificationIndex -> m a) -> m a
+    -- The boolean flag indicates whether the state has been semantically
```

Comment:
Could you elaborate on what `semantically modified` means exactly?

Reply:
Or perhaps not introduce the new term. Just state that the actual bytes in the state have been changed.

Reply:
I have revised.